src/entity/FeedSource.ts

```diff
@@ -12,6 +12,9 @@ export class FeedSource {
   @Index()
   sourceId: string;
 
+  @Column({ default: true })
+  blocked: boolean;
```

Comment:
maybe better to make it enum with `included` and  `excluded` or whatever naming. you want, so we can expand more easily?

Reply:
This follows the tag approach and we thought about it, but you can actually only "block" or "unblock" (neutral meaning removal) so would keep this for now atleast. (as it's only a temporary solution till we get experiment results and we know wheter to go full blown with this)

Reply:
Ok, not really in the loop what it should done.

Reply:
@ilasw do you have frontend PR which uses this, just would like to see how it works as well before merging?

Reply:
Hi @capJavert , yes! You can use is pulling from this branch: https://github.com/dailydotdev/apps/pull/3360 both in source page and in post article.

Reply:
tnx, gonna check it out!